svsim/src/main/scala/verilator/Backend.scala

```diff
@@ -0,0 +1,153 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package svsim.verilator
+
+import svsim._
+import java.io.{BufferedReader, InputStreamReader}
+
+object Backend {
+  object CompilationSettings {
+    sealed trait TraceStyle
+    object TraceStyle {
+      case class Vcd(traceUnderscore: Boolean = false) extends TraceStyle
+    }
+  }
+
+  case class CompilationSettings(
+    traceStyle:                 Option[CompilationSettings.TraceStyle] = None,
+    outputSplit:                Option[Int] = None,
+    outputSplitCFuncs:          Option[Int] = None,
+    disabledWarnings:           Seq[String] = Seq(),
+    disableFatalExitOnWarnings: Boolean = false)
+
+  def initializeFromProcessEnvironment() = {
+    val process = Runtime.getRuntime().exec(Array("which", "verilator"))
```

Comment:
maybe provide `verilator` path in configuration will be better?

Reply:
why? Our usual testing methodology is that users already have any tools (espresso, etc) on their path. I think we should be consistent here